### PR TITLE
Hardware wallet copy updates, referral and tutorial buttons

### PR DESCRIPTION
**Pull request by AlexJupiter**
## Explanation
We need to start adding referral links to our hardware wallet partners, whilst we're doing this we may as well update our tutorial links as well (some of them were super old and broken). I've also included a number of minor copy changes.  

## More Information
I proposed this in Slack and various stakeholders agreed with this approach: https://consensys.slack.com/archives/C026NDN79JB/p1652271544204939 

### Before

https://user-images.githubusercontent.com/7900773/169054132-25b25a10-cd26-428d-973e-d0b2b1b088cf.mov

### After
https://user-images.githubusercontent.com/7900773/169053052-af616d01-4383-40bd-a057-9a7e47cb09ac.mov

## Manual Testing Steps
Go to the "Connect Hardware Wallet" screen to click each hardware wallet section to see all the copy changes and buttons.

## Pre-Merge Checklist

- [ ] PR template is filled out
- [ ] Manual testing complete & passed
- [ ] **IF** this PR fixes a bug, a test that _would have_ caught the bug has been added
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** QA attention is required, "QA Board" label has been applied
- [ ] PR has been added to the appropriate release Milestone

### CODE REVIEW

**Comment by AlexJupiter** [comment]
I have read the CLA Document and I hereby sign the CLA

File: ui/pages/create-account/connect-hardware/select-hardware.js [hunk 1]
```diff
@@ -253,6 +269,22 @@ export default class SelectHardware extends Component {
         {steps.map((step, index) => (
           <div className="hw-connect" key={index}>
             <h3 className="hw-connect__title">{step.title}</h3>
+            <Button
+            className="hw-connect__external-btn-first"
+            type="secondary"
+            onClick={() => window.open("https://gridplus.io/?afmc=7p", "_blank")
```
Comment (owencraston) [inline ui/pages/create-account/connect-hardware/select-hardware.js:275]: same as above. Also what does the _blank do?

Comment (AlexJupiter) [reply ui/pages/create-account/connect-hardware/select-hardware.js:275]: `_blank` should open in a new tab

File: ui/pages/create-account/connect-hardware/select-hardware.js [hunk 2]
```diff
@@ -213,6 +213,22 @@ export default class SelectHardware extends Component {
         {steps.map((step, index) => (
           <div className="hw-connect" key={index}>
             <h3 className="hw-connect__title">{step.title}</h3>
+            <Button
+            className="hw-connect__external-btn-first"
+            type="secondary"
+            onClick={() => window.open("https://shop.ledger.com/?r=17c4991a03fa", "_blank")
+            }
+          >
+           Buy Now
```
Comment (segun) [inline ui/pages/create-account/connect-hardware/select-hardware.js:222]: we should use localization for `Buy Now`

File: ui/pages/create-account/connect-hardware/select-hardware.js [hunk 3]
```diff
@@ -213,6 +213,22 @@ export default class SelectHardware extends Component {
...
+          </Button>
+          <Button
+            className="hw-connect__external-btn"
+            type="secondary"
+            onClick={() => window.open("https://support.ledger.com/hc/en-us/articles/4404366864657-Set-up-and-use-MetaMask-to-access-your-Ledger-Ethereum-ETH-account?docs=true", "_blank")
+            }
+          >
+           Tutorial
```
Comment (segun) [inline ui/pages/create-account/connect-hardware/select-hardware.js:230]: Use L18N here too for `Tutorial`

File: ui/pages/create-account/connect-hardware/select-hardware.js [hunk 4]
```diff
@@ -253,6 +269,22 @@ export default class SelectHardware extends Component {
         {steps.map((step, index) => (
           <div className="hw-connect" key={index}>
             <h3 className="hw-connect__title">{step.title}</h3>
+            <Button
+            className="hw-connect__external-btn-first"
+            type="secondary"
+            onClick={() => window.open("https://gridplus.io/?afmc=7p", "_blank")
+            }
+          >
+           Buy Now
```
Comment (segun) [inline ui/pages/create-account/connect-hardware/select-hardware.js:278]: use L18N for `Buy Now`

File: ui/pages/create-account/connect-hardware/select-hardware.js [hunk 5]
```diff
@@ -253,6 +269,22 @@ export default class SelectHardware extends Component {
...
+          </Button>
+          <Button
+            className="hw-connect__external-btn"
+            type="secondary"
+            onClick={() => window.open("https://docs.gridplus.io/setup/metamask", "_blank")
+            }
+          >
+           Tutorial
```
Comment (segun) [inline ui/pages/create-account/connect-hardware/select-hardware.js:286]: use L18N for `Tutorial`

File: ui/pages/create-account/connect-hardware/select-hardware.js [hunk 6]
```diff
@@ -293,6 +325,22 @@ export default class SelectHardware extends Component {
         {steps.map((step, index) => (
           <div className="hw-connect" key={index}>
             <h3 className="hw-connect__title">{step.title}</h3>
+            <Button
+            className="hw-connect__external-btn-first"
+            type="secondary"
+            onClick={() => window.open("https://shop.trezor.io/product/trezor-one-black?offer_id=35&aff_id=11009", "_blank")
+            }
+          >
+           Buy Now
```
Comment (segun) [inline ui/pages/create-account/connect-hardware/select-hardware.js:334]: use L18N for `Buy Now`

File: ui/pages/create-account/connect-hardware/select-hardware.js [hunk 7]
```diff
@@ -293,6 +325,22 @@ export default class SelectHardware extends Component {
...
+          </Button>
+          <Button
+            className="hw-connect__external-btn"
+            type="secondary"
+            onClick={() => window.open("https://wiki.trezor.io/Apps:MetaMask", "_blank")
+            }
+          >
+           Tutorial
```
Comment (segun) [inline ui/pages/create-account/connect-hardware/select-hardware.js:342]: use L18N for `Tutorial`

File: ui/pages/create-account/connect-hardware/select-hardware.js [hunk 8]
```diff
@@ -318,54 +366,51 @@ export default class SelectHardware extends Component {
...
-              {this.context.t('keystoneTutorial')}
-            </a>
+            <p className="hw-connect__QR-subtitle">Keystone</p>
+            <Button
+            className="hw-connect__external-btn-first"
+            type="secondary"
+            onClick={() => window.open("https://shop.keyst.one/?rfsn=6088257.656b3e9&utm_source=refersion&utm_medium=affiliate&utm_campaign=6088257.656b3e9", "_blank")
+            }>
+              Buy Now
+            </Button>
```
Comment (segun) [inline ui/pages/create-account/connect-hardware/select-hardware.js:376]: use L18N for `Buy Now`

File: ui/pages/create-account/connect-hardware/select-hardware.js [hunk 9]
```diff
@@ -318,54 +366,51 @@ export default class SelectHardware extends Component {
...
+            <Button
+              className="hw-connect__external-btn"
+              type="secondary"
+              onClick={() => window.open("https://support.keyst.one/3rd-party-wallets/eth-and-web3-wallets-keystone/bind-metamask-with-keystone", "_blank")
+              }>
+              Tutorial
```
Comment (segun) [inline ui/pages/create-account/connect-hardware/select-hardware.js:382]: use L18N for `Tutorial`

File: ui/pages/create-account/connect-hardware/select-hardware.js [hunk 10]
```diff
@@ -318,54 +366,51 @@ export default class SelectHardware extends Component {
...
-            >
-              {this.context.t('airgapVaultTutorial')}
-            </a>
+            <p className="hw-connect__QR-subtitle">AirGap Vault</p>
+            <Button
+            className="hw-connect__external-btn-first"
+            type="secondary"
+            onClick={() => window.open("https://airgap.it/", "_blank")
+            }>
+              Download Now
```
Comment (segun) [inline ui/pages/create-account/connect-hardware/select-hardware.js:396]: use L18N for `Download Now`

File: ui/pages/create-account/connect-hardware/select-hardware.js [hunk 11]
```diff
@@ -318,54 +366,51 @@ export default class SelectHardware extends Component {
...
+            </Button>
+            <Button
+              className="hw-connect__external-btn"
+              type="secondary"
+              onClick={() => window.open("https://support.airgap.it/guides/metamask/", "_blank")
+              }>
+              Tutorial
```
Comment (segun) [inline ui/pages/create-account/connect-hardware/select-hardware.js:403]: use L18N for `Tutorial`

**Review by segun — COMMENTED** [review]
I left a bunch of comments on where we need to apply localization.

**Comment by AlexJupiter** [comment]
The only changes that are relevant here are those in `ui/pages/create-account/connect-hardware/select-hardware.js` and `app/_locales/en/messages.json`. 

All those other changes are because I kept having local build problems with various permutations of https://github.com/MetaMask/metamask-extension/issues/14451 and https://github.com/MetaMask/metamask-extension/issues/10748 (even though I'm on MacOS) being noticed. I attempted to fix but failed so I can't run this on my local machine right now.

**Comment by ryanml** [comment]
@segun @owencraston this is ready for another pass, thanks!

**Review by segun — APPROVED** [review]

**Review by adonesky1 — APPROVED** [review]
LGTM

**Pull request merged**